packages/editor/src/components/inserter-list-item/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { normalizeIconObject } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import BlockIcon from '../block-icon';
+
+function InserterListItem( {
+	icon,
+	hasChildBlocksWithInserterSupport,
+	onClick,
+	isDisabled,
+	title,
+	className,
+	...props
+} ) {
+	icon = normalizeIconObject( icon );
+
+	const itemIconStyle = icon ? {
+		backgroundColor: icon.background,
+		color: icon.foreground,
+	} : {};
+	const itemIconStackStyle = icon && icon.shadowColor ? {
+		backgroundColor: icon.shadowColor,
+	} : {};
+
+	return (
+		<li className="editor-block-types-list__list-item">
+			<button
```

Comment:
Since we move this code, can we use [Button](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/button) component instead of HTML element?